KMB/make_KMB_info.py

```diff
@@ -154,22 +197,21 @@ def generate_content_cats(self, item):
         :param item: the KMBItem to analyse
         :return: list of categories (without "Category:" prefix)
         """
-        cats = []
+        cats = item.content_cats
 
         # depicted
         if item.fmis:
-            cats.append('Archaeological monuments in %s' % item.landskap)
-            cats.append('Archaeological monuments in %s County' % item.lan)
+            cats.add('Archaeological monuments in %s' % item.landskap)
+            cats.add('Archaeological monuments in %s County' % item.lan)
```

Comment:
That is [T164719](https://phabricator.wikimedia.org/T164719). Note however that the WLM bot does some categorisation if it is in the top category but not otherwise.